### PR TITLE
Add instructions for choosing pip version

### PR DESCRIPTION
**Pull request by alexmorley**
## Thanks for contributing a pull request!

If you are contributing for the first time or provide a trivial fix don't worry too
much about the checklist - we will help you get started.

## Contribution checklist:

(also see [CONTRIBUTING.rst](../tree/master/CONTRIBUTING.rst) for details)

- [ ] wrote descriptive pull request text
- [ ] added/updated test(s)
- [ ] updated/extended the documentation
- [ ] added relevant [issue keyword](https://help.github.com/articles/closing-issues-using-keywords/)
      in message body
- [ ] added news fragment in [changelog folder](../tree/master/docs/changelog)
  * fragment name: `<issue number>.<type>.rst` for example (588.bugfix.rst)
  * `<type>` is must be one of `bugfix`, `feature`, `deprecation`,`breaking`, `doc`, `misc`
  * if PR has no issue: consider creating one first or change it to the PR number after creating the PR
  * "sign" fragment with "by :user:`<your username>`"
  * please use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files - by :user:`superuser`."
  * also see [examples](../tree/master/docs/changelog)
- [ ] added yourself to `CONTRIBUTORS` (preserving alphabetical order)

### CODE REVIEW

File: docs/config.rst [hunk 1]
```diff
@@ -497,7 +497,8 @@ Complete list of settings that you can put into ``testenv*`` sections:
     .. versionadded:: 3.10
 
     Set to ``true`` if you want virtualenv to upgrade pip/wheel/setuptools to
-    the latest version.
+    the latest version. When this is set you may add e.g. ``VIRTUALENV_PIP=20.3.3``
```
Comment (gaborbernat) [inline docs/config.rst:500]: I believe this should work without this version too, not?

Comment (alexmorley) [reply docs/config.rst:500]: Not for everyone apparently e.g. #1768. That's why I put `you may add`, but maybe that isn't clear?

Regardless if you want any specific version that's not exactly latest you do need the flag.

Comment (alexmorley) [reply docs/config.rst]: maybe its clearer now?

**Review by gaborbernat — APPROVED** [review]

**Pull request merged**